src/App.tsx

```diff
@@ -1,167 +1,53 @@
-import { Loader } from './components/Loader';
 
+import classNames from 'classnames';
+import { NavLink, Navigate, Route, Routes } from 'react-router-dom';
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
-
-    <main className="section">
-      <div className="container">
-        <h1 className="title">Home Page</h1>
-        <h1 className="title">People Page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="block">
-          <div className="box table-container">
-            <Loader />
-
-            <p data-cy="peopleLoadingError" className="has-text-danger">
-              Something went wrong
-            </p>
-
-            <p data-cy="noPeopleMessage">There are no people on the server</p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
+import { HomePage } from './components/Loader/HomePage/HomePage';
+import { PeoplePage } from './components/Loader/PeoplePage/PeoplePage';
+
+
+
+const activeLink = ({ isActive }: { isActive: boolean }) =>
```

Comment:
```suggestion
const getActiveClass = ({ isActive }: { isActive: boolean }) =>
```
It's not a link, and function names should start from the verb